src/util/createInstantSearch.js

```diff
@@ -0,0 +1,115 @@
+import instantsearch from 'instantsearch.js/es';
+import algoliaHelper from 'algoliasearch-helper';
+const { SearchParameters, SearchResults } = algoliaHelper;
+import { _objectSpread } from './polyfills';
+import { warn } from './warn';
+
+export const createInstantSearch = ({ searchClient, indexName, options }) => {
```

Comment:
Why do we change the signature of `instantsearch`? Is there a reason we want to scope `options`?

Reply:
TODO: make signature the same as InstantSearch.js